rollup.config.js

```diff
@@ -219,10 +219,10 @@ function remixDev() {
     },
     {
       external: (id) => isBareModuleId(id),
-      input: [`${sourceDir}/cli/codemod/run/transforms/index.ts`],
+      input: [`${sourceDir}/cli/migrate/run/transforms/index.ts`],
       output: {
-        banner: createBanner("@remix-run/codemod", version),
-        dir: `${outputDir}/cli/codemod/run/transforms`,
+        banner: createBanner("@remix-run/migrate", version),
```

Comment:
This should have been done in my PR as well.
Probably an oversight when copy/pasting the config I had for the separate package 🙈

```suggestion
        banner: createBanner("@remix-run/dev", version),
```

Reply:
Unfortunately they both are necessary, as transforms aren't used directly (they're used by their filePath).

This extra Rollup entry is getting the transforms index, which is exporting all transforms (since they're used in the `transformFunctionByName` object in `migrate/run/index.ts`), so is also exporting the escape-hatch, which is why the transform code is included in the build.

Was looking into doing it with a glob (something like `${sourceDir}/cli/migrate/run/transforms/**/jscode-transform/**` as `input`) so we don't need the escape-hatch in the code, but couldn't find how to do so.